pkg/debugapi/export_test.go

```diff
@@ -16,10 +16,14 @@ type (
 	WelcomeMessageResponse   = welcomeMessageResponse
 	BalancesResponse         = balancesResponse
 	BalanceResponse          = balanceResponse
+	SettlementResponse       = settlementResponse
+	SettlementsResponse      = settlementsResponse
 )
 
 var (
-	ErrCantBalance   = errCantBalance
-	ErrCantBalances  = errCantBalances
-	ErrInvaliAddress = errInvaliAddress
+	ErrCantBalance     = errCantBalance
+	ErrCantBalances    = errCantBalances
+	ErrCantSettlement  = errCantSettlements
+	ErrCantSettlements = errCantSettlements
```

Comment:
Something is not right here, you use the same value for both `ErrCantSettlement` and `ErrCantSettlements`.